cli/cmd/compose/version.go

```diff
@@ -34,9 +34,10 @@ type versionOptions struct {
 func versionCommand() *cobra.Command {
 	opts := versionOptions{}
 	cmd := &cobra.Command{
-		Use:   "version",
-		Short: "Show the Docker Compose version information",
-		Args:  cobra.MaximumNArgs(0),
+		Use:    "version",
+		Short:  "Show the Docker Compose version information",
+		Args:   cobra.MaximumNArgs(0),
+		Hidden: true,
```

Comment:
Why would we want to hide this command?

Reply:
Should be hidden when used inside compose-cli but not for cli-plugin ...